src/Microsoft.Health.Api/Features/HealthChecks/HealthCheckCachingOptions.cs

```diff
@@ -17,19 +16,22 @@ public class HealthCheckCachingOptions
         /// <summary>
         /// Gets or sets the amount of time for which an instance of <see cref="HealthCheckResult"/> is considered valid.
         /// </summary>
-        [Range(typeof(TimeSpan), "00:00:00", "1.00:00:00", ConvertValueInInvariantCulture = true, ParseLimitsInInvariantCulture = true)]
         public TimeSpan Expiry { get; set; } = TimeSpan.FromSeconds(1);
 
         /// <summary>
         /// Gets or sets the amount of time before the expiry for which the instance of <see cref="HealthCheckResult"/>
         /// should be refreshed.
         /// </summary>
-        [Range(typeof(TimeSpan), "00:00:00", "1.00:00:00", ConvertValueInInvariantCulture = true, ParseLimitsInInvariantCulture = true)]
         public TimeSpan RefreshOffset { get; set; } = TimeSpan.Zero;
 
         /// <summary>
         /// Gets or sets a flag that indicates whether <see cref="HealthStatus.Unhealthy"/> should be cached.
         /// </summary>
         public bool CacheFailure { get; set; } = true;
```

Comment:
Do we need to allow this option? Should'nt we always cache failure? We don't want to retry a failed sql ops immediately either.

Reply:
Removed